product_docs/docs/biganimal/release/using_cluster/02_connecting_your_cluster/01_connecting_from_azure/index.mdx

```diff
@@ -51,108 +53,94 @@ To walk through an example in your own environment, you need:
 -   The IP address of your cluster. You can find the IP address of your cluster using the following command:
 
     ```shell
-    dig +short p-mckwlbakq5.brcxzr08qr7rbei1.biganimal.io 
+    dig +short p-mckwlbakq5.brcxzr08qr7rbei1.biganimal.io
     __OUTPUT__
     10.240.1.218
     ```
--   A Postgresql client, such as [psql](https://www.postgresql.org/download/), installed on your client VM.
+-   A PostgreSQL client, such as [psql](https://www.postgresql.org/download/), installed on your client VM.
 
-#### Step 1: Create an Azure Private Link service for your cluster
+#### Step 1: BigAnimal automatically provision an Azure Private Link Service for every private Postgres cluster
```

Comment:
```suggestion
BigAnimal automatically provisions an Azure Private Link Service for every private Postgres cluster
```
This isn't a step like the other steps, which tell a user what to do. I would change Step 2 to Step 1 and add this and the following sentence as a note under the new Step 1. I can make this change for you, if you agree.